demo/app/android/app/src/androidTest/kotlin/integration_test.kt

```diff
@@ -3,20 +3,15 @@ import androidx.test.filters.SmallTest
 import androidx.test.platform.app.InstrumentationRegistry
 import com.google.common.truth.Truth.assertThat
 import dev.trustbloc.wallet.BuildConfig
-import dev.trustbloc.wallet.sdk.api.CreateDIDOpts
 import dev.trustbloc.wallet.sdk.api.KeyWriter
 import dev.trustbloc.wallet.sdk.api.VerifiableCredentialsArray
-import dev.trustbloc.wallet.sdk.credential.CredentialsOpt
+import dev.trustbloc.wallet.sdk.credential.CredentialsArg
```

Comment:
I see other options ending with plural (`zzzArgs`), and this one is singular (`CredentialsArg`). What's the reason for this?

Reply:
Also, I see 2 suffices - zzzRequiredArgs and zzzOptionalArgs. This one match to that standards.

Reply:
I only put `required` in the name for a few places where there are many different required input arguments (OpenID4CI/OpenID4VP), but really any name could do. I think a better solution in the future would be to try to defer passing in those extra arguments until they're really needed, and that would make those big argument bundles less necessary.